### ISSUE TITLE
Configuring pkgtags_url is disasterous for mashes

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/bodhi#1634]
We [learned today](https://github.com/fedora-infra/bodhi/pull/1633) that setting the configuration for ```pkgtags_url``` to any "truthy" value causes a [severe bug](https://bugzilla.redhat.com/show_bug.cgi?id=1463561) for Fedora and EPEL. According to @puiterwijk, the pkgtag feature has never worked correctly and it is recommended that the feature be killed.

The code that does this lives in [```bodhi.server.metadata.ExtendedMetadata.insert_pkgtags()```](https://github.com/fedora-infra/bodhi/blob/2.8.0/bodhi/server/metadata.py#L304-L317):

```
    def insert_pkgtags(self):
        """Download and inject the pkgtags sqlite from fedora-tagger"""
        if config.get('pkgtags_url'):
            try:
                tags_url = config.get('pkgtags_url')
                tempdir = tempfile.mkdtemp('bodhi')
                local_tags = os.path.join(tempdir, 'pkgtags.sqlite')
                log.info('Downloading %s' % tags_url)
                urlgrab(tags_url, filename=local_tags)
                self.modifyrepo(local_tags)
            except:
                log.exception("There was a problem injecting pkgtags")
            finally:
                shutil.rmtree(tempdir)
```

Since this code has never worked and since it is very bad if it is run, I think it's reasonable to delete it without [semantic versioning](http://semver.org/) concerns, but we can call a release with this change Bodhi 3 if someone feels strongly about it. I personally see it as a bug fix since it's removing a dangerous bit of code.

**Comment by bowlofeggs** [comment]
Deleting this code will have a nice side effect of raising Bodhi's test coverage, since this code isn't covered ☺

**Comment by bowlofeggs** [comment]
The patch for this issue should be included in the upcoming 2.10.0 release:

https://github.com/fedora-infra/bodhi/pull/1744
